README.md

```diff
@@ -1,2 +1,30 @@
-# Multilingual_NER
-Applying BERT to named entity recognition in English and Russian.
+# multilingual_NER
+
+## About
+Deployable [BERT](https://github.com/google-research/bert) ensemble models for named entity recognition in English and Russian. `multilingual_NER` provides a toolkit to train, validate, and deploy NER models with an interactive frontend and callable API.
+
+## Installation
```

Comment:
Installation should also include links to datasets and instructions on where to download to (for any data-sets too large to fit into the repo)

Reply:
So far, none of the datasets are too large - down the line, should they be removed anyway and replaced with shell scripts to download them?

Reply:
notes from meeting: "This is a nice to have - not necessary for a small project, but does look good for production"